docker-compose.yml

```diff
@@ -8,16 +8,13 @@ services:
     ports:
       - "6379:6379"
   db:
-    image: postgres:latest
+    image: postgres:12-alpine
     restart: always
     environment:
       - POSTGRES_DB=db
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=postgres
+    expose:
+      - 5432
     ports:
       - '5432:5432'
-    volumes:
-      - db:/var/lib/postgresql/data
-volumes:
-  db:
-    driver: local
```

Comment:
Why are we able to drop those?

Reply:
Hm, this looks like an accidental commit. But I'd ask the reverse question: why were they there in the first place?

`driver: local` is the default.

And why map the data directory? To keep data between postgres container restarts? Seems odd in a dev environment, but maybe you actually want it to work this way? Then I'd revert.

Reply:
> why were they there in the first place?

That is a good question but I assume that was the reason. But I would agree that especially for the Erica service we do not need to keep data between restarts and it is probably more confusing that we have it.